README.md

```diff
@@ -661,6 +661,9 @@ class TritonPythonModel:
 A complete example for sync and async BLS in Python backend is included in the
 [Examples](#examples) section.
 
+Note: Async BLS is not supported on Python 3.6 or lower due to the async keyword
```

Comment:
Small nit: use `async` instead of async.

Reply:
Good catch